opfpush/src/main/java/org/onepf/opfpush/model/OPFError.java

```diff
@@ -16,21 +16,30 @@
 
 package org.onepf.opfpush.model;
 
+import org.onepf.opfpush.listener.EventListener;
+
 /**
- * Error of registration or unregistration.
- * Divided on two categories: <b>recoverable</b> and <b>non recoverable</b>.
+ * Error of registration or unregistration operation.
```

Comment:
"Error of registration or unregistration operation" => "Registration or unregistration operation error"